pom.xml

```diff
@@ -50,7 +50,7 @@
         <vault.user>admin</vault.user>
         <vault.password>admin</vault.password>
         <core.wcm.components.version>2.17.2</core.wcm.components.version>
-        <core.cif.components.version>2.0.0</core.cif.components.version>
+        <core.cif.components.version>2.0.1-SNAPSHOT</core.cif.components.version>
```

Comment:
I marked this PR as do not merge due to this snapshot dependency. We can merge it during the release.

Reply:
The PR can be merged normally. The `dev` branch provides this snapshot dependency via CircleCI. During the release process the snapshot dependency has to be replaced by the newly released version.